weblab/datasets/urls.py

```diff
@@ -0,0 +1,40 @@
+from django.conf.urls import url
+
+from . import views
+
+
+_COMMIT = r'(?P<sha>[^^~:/ ]+)'
```

Comment:
We shouldn't need the `_COMMIT` but the others will probably form part of the detail view url.